src/harness/testcases/registration_testcase.py

```diff
@@ -147,3 +147,69 @@ def test_10_3_4_2_5_3(self):
     # Check registration response
     self.assertFalse('cbsdId' in response)
     self.assertEqual(response['response']['responseCode'], 103)
+
+  @winnforum_testcase
+  def test_WINFF_FT_S_REG_26(self):
+    """Category Error in Array request (responseCode 202)
+
+    The response should be SUCCESS for the first CBSD,
+    FAILURE for the second, third, and fourth CBSDs.
+    """
+
+    device_1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_3 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    device_4 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+
+    # Set up unique FCC IDs
```

Comment:
I wonder if it makes sense to define device_c.json and device_d.json so that we don't have to jump through these hoops? We will probably need them for other tests anyway.